cypress/plugins/index.js

```diff
@@ -0,0 +1,18 @@
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+module.exports = () => {
+	// module.exports = (on, config) => {
+	// `on` is used to hook into various events Cypress emits
+	// `config` is the resolved Cypress config
+};
```

Comment:
Do we need this file?

Reply:
It is auto generated, so I think it would be better to keep it.

Reply:
Why not add it to `.gitignore` and add it to vcs when necessary (if ever)?

Reply:
sounds good to me 👍